modules/preview-react/form-field/lib/FormFieldLabel.tsx

```diff
@@ -32,29 +31,59 @@ const asteriskStyles = createStyles({
   color: brand.error.base,
 });
 
-const labelModifiers = createModifiers({
-  orientation: {
-    horizontal: createStyles({
-      float: 'left',
-      maxHeight: space.xl,
-    }),
-    vertical: createStyles({
-      width: 'auto',
-    }),
+const labelStencil = createStencil({
+  base: {
+    ...system.type.subtext.large,
+    fontWeight: system.fontWeight.medium,
+    color: base.blackPepper300,
+    paddingInlineStart: 0,
+    marginBottom: space.xxxs,
```

Comment:
I guess there should be `system.space` tokens instead old tokens 🤔

Reply:
Next styling PR will fix this!

Reply:
But as you add it here wouldn't it be better to already have it? Also, almost all other styles here is new tokens

Reply:
fair.. i was being lazy :)